src/components/SlideElementWrapper.js

```diff
@@ -1,38 +1,45 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { useSpring, animated } from 'react-spring';
 
 import useSlide, { SlideContext } from '../hooks/useSlide';
 
 /**
+ * SlideElementWrapper provides a component for animating slideElements
+ * Anything wrapped inside will be affected by the transition.
  *
- *
+ * It is currently using useSpring but ideally we will be able to switch
+ * to whatever react-spring hook a user desires!
  */
 
-const SlideElementWrapper = props => {
-  const { elementNum, transitionEffect, children } = props;
+const SlideElementWrapper = ({ elementNum, transitionEffect, children }) => {
   const [state] = React.useContext(SlideContext);
 
-  const [styleProps, set] = useSpring(() =>
-    transitionEffect
-      ? console.log(transitionEffect.from) || transitionEffect.from
-      : { opacity: 0 }
-  );
+  const [styleProps, set] = useSpring(() => transitionEffect.from);
 
   React.useEffect(() => {
     if (state && elementNum <= state.currentSlideElement) {
-      if (transitionEffect) {
-        console.log(transitionEffect.to);
-        set({
-          to: transitionEffect.to
-        });
-      }
-      set({
-        opacity: 1
-      });
+      set({ from: transitionEffect.from, to: transitionEffect.to });
```

Comment:
Could we simplify this even further and drop the `from` case and just do `set({ transitionEffect.to })`? Seems to produce the same result I think.